### PR TITLE
Update documentation regarding topology_bin in the SubmitterMain

### PR DESCRIPTION
**Pull request by srkukarni**
Cli passes only the file name of the topology_bin.
Thus it's not necessary for the schedulers to get the basename.

### CODE REVIEW

**Comment by objmagic** [comment]
Also [here](https://github.com/twitter/heron/blob/master/heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java#L206), which is used by other scheduler indirectly.

**Comment by objmagic** [comment]
also please fix style error in CI.

File: heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java [hunk 1]
```diff
@@ -189,7 +189,7 @@ private static Options constructOptions() {
         .build();
 
     Option topologyJar = Option.builder("j")
-        .desc("user heron topology jar/pex file path")
+        .desc("user heron topology jar/tar/pex file")
         .longOpt("topology_bin")
         .hasArgs()
         .argName("topology binary file")
```
Comment (billonahill) [inline heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java]: Is there a way to covey that this doesn't actually represent a file or path on the local disk, but the name of the file that will be invoked on the executor?

**Comment by srkukarni** [comment]
@objmagic We cannot remove that since a) that's topology defn file and b) scheduler currently needs to read topology_defn. 
Fixed style error.

**Comment by objmagic** [comment]
@srkukarni yes you are right.

File: heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java (hunk 1 above)
Comment (srkukarni) [inline heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java]: Suggestions please?

Comment (billonahill) [inline heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java]: .desc("The remote filename of the heron topology jar/tar/pex file to be run by the executor.")
.argName("topology binary filename on the cluster")

**Comment by srkukarni** [comment]
Made the changes wrt arg description based on feedback. Please take a look again. Thanks!

**Pull request merged**